cloud/kubernetes/helm/templates/_helpers.tpl

```diff
@@ -45,7 +43,7 @@ volumeMounts:
 volumes:
 {{- end }}
 {{- range $i, $vName := tuple "data-store" "logs" "rms-shared" "rms-security" "rms-webstudio" }}
```

Comment:
can we delete rms-security?

Also the same range tuples with same values were added in many places, can we move the range values to values.yaml

Reply:
NO. rms-security folder need to be mounted.

Values.yaml should be used to collect flags/data from the user while installing the chart. Not a good idea to move this to values.yaml